features/support/env.rb

```diff
@@ -57,3 +57,21 @@ def skip_below(os, version)
 Before('@stress_test') do |_scenario|
   skip_this_scenario('Skipping: Run is not configured for stress tests') if ENV['STRESS_TEST'].nil?
 end
+
+Maze.hooks.after do |scenario|
+  if Maze.driver.capabilities['platformName'] == 'iOS'
+    data = Maze.driver.pull_file '@com.bugsnag.iOSTestApp/Documents/kscrash.log'
+
+    folder1 = File.join(Dir.pwd, 'maze_output')
+    folder2 = scenario.failed? ? 'failed' : 'passed'
+    folder3 = scenario.name.gsub(/[:"& ]/, "_").gsub(/_+/, "_")
+
+    path = File.join(folder1, folder2, folder3)
+
+    FileUtils.makedirs(path)
+
+    File.open(File.join(path, 'kscrash.log'), 'wb') { |file| file << data }
+  end
```

Comment:
We should probably provide an abstraction in Maze to allow clients to add their own log files, but this seems fine for now.

Reply:
Ideally Maze would expose a way to get the path to store log files in. One to consider if any other clients need this functionality!